I'm sorry, but an uncaught exception occurred.

After initialization, but before game start.
  File "renpy/common/00start.rpy", line 76, in _init_language
    renpy.change_language(language)
  File "game/screens.rpy", line 154, in <module>
    xpos gui.dialogue_xpos
AttributeError: 'StoreModule' object has no attribute 'dialogue_xpos'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\main.py", line 487, in main
    run(restart)
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\main.py", line 69, in run
    renpy.translation.init_translation()
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\translation\__init__.py", line 506, in init_translation
    renpy.store._init_language()  # @UndefinedVariable
  File "renpy/common/00start.rpy", line 76, in _init_language
    renpy.change_language(language)
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\translation\__init__.py", line 583, in change_language
    new_change_language(tl, language)
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\translation\__init__.py", line 553, in new_change_language
    i.apply()
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\ast.py", line 2224, in apply
    value = renpy.python.py_eval(expr)
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\python.py", line 1749, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\David\Documents\Ren'py\renpy-6.99.12.4-sdk\renpy\python.py", line 1743, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/screens.rpy", line 154, in <module>
    xpos gui.dialogue_xpos
AttributeError: 'StoreModule' object has no attribute 'dialogue_xpos'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Bar Story 1.1
